packages/runtime-dom/src/modules/props.ts

```diff
@@ -95,7 +95,7 @@ export function patchDOMProp(
   // some properties has getter, no setter, will error in 'use strict'
   // eg. <select :type="null"></select> <select :willValidate="null"></select>
   try {
-    el[key] = value
+    el[camelize(key)] = value
```

Comment:
Shouldn't do it here as it affects all elements. This should only apply to Vue custom elements. See ea3efa09e (reused test from this PR)